assets/js/membraneWebRTC.ts

```diff
@@ -483,6 +630,7 @@ export class MembraneWebRTC {
   private onRemoteCandidate = (candidate: RTCIceCandidate) => {
     try {
       const iceCandidate = new RTCIceCandidate(candidate);
+      if (iceCandidate.sdpMLineIndex == null || iceCandidate.sdpMid == null) return;
```

Comment:
I would remove this as we will not see errors when server generates wrong candidates